### PR TITLE
Add supported operations in readme

### LINKED ISSUE #59: [DOC] How to invoke function in python provider?
Do we have any doc to describe how to invoke function in python provider like following function?
https://github.com/Microsoft/omi-script-provider/blob/master/samples/omi_tester/omi_tester_schema.mof#L5

For current scope, we will support enumerate, modify, create, get, delete and invoke operation for python provider, right?

I am not sure if invoke support in this release or not.

### PR DESCRIPTION
**Pull request by JumpingYang001**
Fixed #59, thanks.
@EMumau @alinbalutoiu @c64cosmin , could you to review this PR? thanks very much!

### CODE REVIEW

**Review by c64cosmin — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -88,6 +88,81 @@ We can now register the Python Provider using the omireg command:
...
+    IPAddress=ff02::1
+}
+instance of Hosts
+{
+    [Key] FQDN=ip6-allrouters
+    IPAddress=ff02::2
+}
+```
+
+create operation:
```
Comment (c64cosmin) [inline README.md]: Minor change, should be in bold or with bullet.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,81 @@ We can now register the Python Provider using the omireg command:
...
+```
+/opt/omi/bin/omicli ci root/cimv2 { Hosts FQDN 'omi64-cent7-01.scx.com' IPAddress '10.226.158.99' }
+```
+
+modify operation:
```
Comment (c64cosmin) [inline README.md]: same ^

File: README.md [hunk 3]
```diff
@@ -88,6 +88,81 @@ We can now register the Python Provider using the omireg command:
...
+```
+/opt/omi/bin/omicli mi root/cimv2 { Hosts FQDN 'omi64-cent7-01.scx.com' IPAddress '10.226.158.100' }
+instance of Hosts
+{
+    [Key] FQDN=omi64-cent7-01.scx.com
+    IPAddress=10.226.158.100
+}
+```
+
+get operation:
```
Comment (c64cosmin) [inline README.md]: same ^

File: README.md [hunk 4]
```diff
@@ -88,6 +88,81 @@ We can now register the Python Provider using the omireg command:
...
+```
+/opt/omi/bin/omicli gi root/cimv2 { Hosts FQDN 'omi64-cent7-01.scx.com' }
+instance of Hosts
+{
+    [Key] FQDN=omi64-cent7-01.scx.com
+    IPAddress=10.226.158.100
+}
+```
+
+delete operation:
```
Comment (c64cosmin) [inline README.md]: same ^

File: README.md [hunk 5]
```diff
@@ -88,6 +88,81 @@ We can now register the Python Provider using the omireg command:
...
+```
+/opt/omi/bin/omicli di root/cimv2 { Hosts FQDN 'omi64-cent7-01.scx.com' }
+```
+
+invoke operation:
```
Comment (c64cosmin) [inline README.md]: same ^

**Review by c64cosmin — APPROVED** [review]

**Comment by JumpingYang001** [comment]
will update, thanks for reviewing!

**Pull request merged**